suse_openstack_cloud

```diff
@@ -396,13 +396,12 @@ fi
 section_header "RabbitMQ broker"
 
 if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+    rabbitmq_var_log="/var/log/rabbitmq"
+    rabbitmq_report_log=$rabbitmq_var_log/rabbitmq_report.log
```

Comment:
using *.log means that it will get logrotated though.. perhaps not use the suffix?